teamcity/main.tf

```diff
@@ -0,0 +1,51 @@
+provider "aws" {
+  region = "${var.AWS_REGION}"
+}
+
+module "dcos" {
+  source  = "dcos-terraform/dcos/aws"
+  version = "~> 0.2.10"
+
+  cluster_name        = "tf-ci-${var.teamcity.build.id}-"
+  cluster_name_random_string = true
+
+  ssh_public_key_file = "id_rsa.pub"
+  admin_ips           = ["0.0.0.0/0"]
```

Comment:
How about restricting access to teamcity node
https://github.com/dcos/dcos/blob/master/test_util/main.tf#L102

Reply:
done